bin/oneoff/backfill_parent-child_relationships.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
```

Comment:
I think this script will be run manually on each managed environment? If it's not a scheduled script, we probably don't need the `only_one` logic.

Reply:
that's correct, but it takes a couple minutes to run so I left this in just for safety. Is there any downside to including this?